packages/react/src/menu-content/MenuContent.tsx

```diff
@@ -0,0 +1,23 @@
+import * as RadixMenu from "@radix-ui/react-dropdown-menu";
+import { forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+import * as styles from "./MenuContent.css";
+
+type MenuContentProps = BoxProps<typeof RadixMenu.Content>;
+
+export const MenuContent = forwardRef<HTMLDivElement, MenuContentProps>(
+  ({ children, className, ...props }, ref) => {
```

Comment:
If we dont use the className in the component, we should omit it. 

```suggestion
  ({ children, ...props }, ref) => {
```